src/utils/gulp/gulp-tasks-test.js

```diff
@@ -17,38 +21,47 @@ module.exports = function(gulp, options) {
   };
 
   gulp.task('test', function(done) {
-    if (options.testPaths) {
+    if (test.paths) {
       var jsxCoverage = require('gulp-jsx-coverage');
       var mocha = require('gulp-mocha');
       var watch = require('gulp-watch');
       var argv = require('yargs').argv;
 
+      // customBabelRc object, or path to local babelrc
+      var customBabelConfig = test.customBabelConfig?
+        test.customBabelConfig :
+        typeof test.pathToBabelRc === 'string'?
+        JSON.parse(require('fs').readFileSync(test.pathToBabelRc, 'utf8')) :
+        {};
+
+      var babelConfig = merge({}, {
+        "presets": [ "es2015", "react" ],
+        "plugins": [
+          "transform-object-rest-spread",
+          "add-module-exports"
+        ]
+      }, customBabelConfig);
+
       jsxCoverage.initModuleLoaderHack(jsxCoverageOptions);
 
-      glob.sync('**src/js/**/*.js').forEach(function (file) {
+      glob.sync(test.globPath || '**src/js/**/*.js').forEach(function (file) {
```

Comment:
sorry i add that string here. But I believe we can safely replace that like with `options.jsAssets`